### PR TITLE
protocol: ext_session_lock added to staging 0.30.x

### PR DESCRIPTION
**Pull request by Shinyzenith**

### CODE REVIEW

**Comment by Shinyzenith** [comment]
I have made sure to update the submodule to the correct hash this time.

File: wayland-server/src/staging.rs [hunk 1]
```diff
@@ -0,0 +1,100 @@
+//! Staging protocols from wayland-protocols
```
Comment (i509VCB) [inline wayland-server/src/staging.rs:1]: I assume this file being added was accidental?

Comment (Shinyzenith) [reply wayland-server/src/staging.rs:1]: Yes...don't know how that got here..

**Review by i509VCB — APPROVED** [review]
Would be nice if the commits could be squashed, but that's a nitpick.

**Comment by Shinyzenith** [comment]
> Would be nice if the commits could be squashed, but that's a nitpick.

like a force commit? I'll google how to do it. 1 Sec.

**Comment by Shinyzenith** [comment]
Done!

**Review by PolyMeilex — APPROVED** [review]

**Comment by Shinyzenith** [comment]
Updated changelog.

**Review by i509VCB — APPROVED** [review]

**Comment by vberger** [comment]
Thank you!

**Pull request merged**